app/policies/acceptance_policy.rb

```diff
@@ -0,0 +1,13 @@
+class AcceptancePolicy < ApplicationPolicy
+  attr_reader :user, :pair_request
+
+  def initialize(user, pair_request)
+    @user = user
+    @pair_request = pair_request
+  end
```

Comment:
shouldn't need this, it should just get inherited.

Reply:
hmm, should we nest the AcceptancePolicy class under PairRequstPolicy? currently we have it under ApplicationPolicy.

Reply:
we really don't use any of the PairRequestPolicy methods

Reply:
under ApplicationPolicy is perfect. This file though should most likely be at `app/policies/pair_request/acceptance_policy.rb`

Reply:
done!